client/reader/stream/reader-sidebar-organizations/index.jsx

```diff
@@ -25,12 +25,15 @@ export class ReaderSidebarOrganizations extends Component {
 	}
 
 	renderItems() {
-		const { path } = this.props;
+		const { path, translate } = this.props;
 		const organization = this.fetchOrganization();
+		const organizationSlug = organization.slug === 'a8c' ? translate( 'a8c' ) : translate( 'p2' );
```

Comment:
I don't think `a8c` or `p2` will translate very well, so I'd probably leave these as is (untranslated).

Briefly checking, I see existing translations leave "P2" as "P2" in other languages, since it's more like a brand.

Reply:
Agreed with @creativecoder. Feel free to declare those as constants outside of the component if that makes it less awkward.